### PR TITLE
Handle alias variables in Susan

### PR DESCRIPTION
**Pull request by sjoelund**

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2018-01-05_08-56-27](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/5073/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2018-01-05_09-46-12](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/5074/).

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2018-01-05_10-37-03](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/5075/console).

File: Compiler/Template/CodegenCFunctions.tpl [hunk 1]
```diff
@@ -4100,10 +4100,14 @@ template crefToCStr(ComponentRef cr, Integer ix, Boolean isPre, Boolean isStart)
...
   then
     'data->simulationInfo->inlineData->algOldVars[<%index%>]'
+  case SIMVAR(aliasvar=ALIAS(varName=varName)) then crefToCStr(varName, ix, isPre, isStart)
+  case SIMVAR(aliasvar=NEGATEDALIAS(varName=varName), type_=T_BOOL()) then '!(<%crefToCStr(varName, ix, isPre, isStart)%>)'
+  case SIMVAR(aliasvar=NEGATEDALIAS(varName=varName)) then '-(<%crefToCStr(varName, ix, isPre, isStart)%>)'
   case SIMVAR(varKind=JAC_VAR())
   case SIMVAR(varKind=JAC_DIFF_VAR())
   case SIMVAR(varKind=SEED_VAR())
   case SIMVAR(varKind=DAE_RESIDUAL_VAR())
+  case SIMVAR(varKind=DAE_RESIDUAL_VAR())
```
[the remark this answers is not in the document]
Comment (sjoelund) [reply Compiler/Template/CodegenCFunctions.tpl]: Copy-pasta I guess. Been writing directly on my laptop.

**Pull request closed without merging**